src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -43,72 +43,128 @@ export default {
             },
             control: { type: "select" },
         },
+        position: {
+            options: Object.values(TooltipPosition),
+            control: { type: "select" },
+            defaultValue: TooltipPosition.Bottom,
+        },
+        alignment: {
+            options: Object.values(TooltipAlignment),
+            control: { type: "select" },
+            defaultValue: TooltipAlignment.Middle,
+        },
+        withArrow: {
+            control: { type: "boolean" },
+            defaultValue: false,
+        },
+        flip: {
+            control: { type: "boolean" },
+            defaultValue: true,
+        },
+        hoverDelay: {
+            control: { type: "number" },
+            defaultValue: 200,
+        },
     },
-} as Meta;
+} as Meta<TooltipProps>;
+
+export const TooltipComponent: Story<TooltipProps> = (args: TooltipProps) => {
+    const triggerRef = useRef<HTMLButtonElement>(null);
+    return (
+        <div className="tw-w-screen tw-h-screen tw-flex tw-justify-center tw-items-center">
+            <button ref={triggerRef} className="tw-flex tw-justify-center tw-items-center">
+                <IconAttentionFilled size={IconSize.Size16} />
+                <span>Tooltip trigger</span>
+            </button>
+            <Tooltip {...args} triggerRefElement={triggerRef} />
+        </div>
+    );
+};
 
-export const Tooltip: Story<TooltipProps> = (args: TooltipProps) => <TooltipComponent {...args} />;
+export const MultipleTooltipsComponent: Story<TooltipProps> = (args: TooltipProps) => {
+    const firstTriggerRef = useRef<HTMLButtonElement>(null);
+    const secondTriggerRef = useRef<HTMLButtonElement>(null);
+    return (
+        <div className="tw-w-screen tw-h-screen tw-flex tw-justify-center tw-items-center">
+            <p className="tw-mr-2">Label</p>
+            <button ref={firstTriggerRef} className="tw-mr-1">
+                <IconAttentionFilled size={IconSize.Size16} />
+            </button>
+            <Tooltip {...args} triggerRefElement={firstTriggerRef} withArrow={true} />
```

Comment:
WithArrow Storybook control is overwritten by this prop and it feels a bit weird that it toggles one and not the other